docs/production-deployment/cloud/get-started/certificates.mdx

```diff
@@ -428,7 +428,7 @@ AuthorizedClientCertificate {
 When using a `*` wildcard, the following values are valid:
 
 - `*.example.com` matches `code.example.com` and `text.example.com`.
-- `Example Code*` matches `Example code` and `Example Code Inc`.
+- `Example Code*` matches `Example Code` and `Example Code Inc`.
```

Comment:
I assume this was a typo. It should not match the lowercase spelling, but am also not sure if it would match this.